### PR TITLE
Splitting out head and main stylesheets

### PR DESCRIPTION
**Pull request by ironsidevsquincy**

### CODE REVIEW

**Comment by andygnewman** [comment]
@ironsidevsquincy did you spot any ways in which we could further split the css to reduce the amount that goes in head vs main?

File: nodemon.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "ignoreRoot": [],
+  "watch": [
+    "node_modules",
```
Comment (bjfletcher) [inline nodemon.json]: handy config - how about putting this config in n-makefile, and watching the whole of public folder?

Comment (matthew-andrews) [inline nodemon.json]: +1 i can help with this if people need

Comment (ironsidevsquincy) [inline nodemon.json]: yep, was thinking that. don't need to watch the whole of `public` as only need to restart the server when the `head` css changes (as we read it off the fs to inject into the head)

**Comment by ironsidevsquincy** [comment]
@andygnewman tbh, think you guys would be better placed to decide what goes where, as you know more about the styling. Does it look ok to you? Basic philosophy is anything that's not on the first screen, or styles JS related things, goes in `main`, otherwise `head`

**Pull request merged**